chains/orchestrator-relays/runtime/dancelight/src/lib.rs

```diff
@@ -1880,6 +1880,7 @@ impl pallet_pooled_staking::Config for Runtime {
     type LeavingRequestTimer = SessionTimer<StakingSessionDelay>;
     type EligibleCandidatesBufferSize = ConstU32<100>;
     type EligibleCandidatesFilter = CandidateHasRegisteredKeys;
```

Comment:
`EligibleCandidatesFilter` still doesn't filter out inactive candidates, so calls to `update_candidate_position` will not remove them. `update_candidate_position` is the one using `EligibleCandidatesFilter` to know if it should appear in the list.

Since `CandidateHasRegisteredKeys` is defined in this file, you can modify it (with rename) to also check if the candidate is offline or not.